Gemfile

```diff
@@ -143,6 +143,9 @@ gem 'dogapi', '~> 1.45'
 # https://github.com/ruby/net-protocol/issues/10
 # This gem can be removed once we upgrade to Ruby 3.1.
 gem 'net-http'
+# Temporarily locking the 'mail' version until the next version of Rails is released
+# https://github.com/rails/rails/pull/46650
+gem 'mail', '~> 2.7.1'
```

Comment:
locked to avoid another login failure

Reply:
Should this be moved above the `Assets-related gems` section?

Reply:
No, the `mail` gem is not related to assets: https://github.com/mikel/mail